misc/docerina/src/main/java/org/ballerinalang/docgen/generator/model/Type.java

```diff
@@ -216,12 +217,14 @@ public static Type fromTypeNode(BLangType type, String currentModule) {
         // If anonymous type substitute the name
         if (typeModel.name != null && typeModel.name.contains("$anonType$")) {
             // if anonymous empty record
-            if (type.type instanceof BRecordType && ((BRecordType) type.type).fields.isEmpty()) {
-                    typeModel.name = type.type.toString();
-                    typeModel.generateUserDefinedTypeLink = false;
+            // if anonymous empty record
```

Comment:
`// if anonymous empty record` can remove one, since the same comment duplicated.

Reply:
Added the changes